libpod/boltdb_state_linux.go

```diff
@@ -0,0 +1,104 @@
+// +build linux
+
+package libpod
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/boltdb/bolt"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func (s *BoltState) setNamespace(ctr *Container, netNSPath string, newState *containerState) error {
```

Comment:
This shouldn't be a BoltDB thing - the logic is generally just for handling network namespaces. Something like `replaceNetworkNamespace(ctr *Container, newNSPath string) error`

Reply:
fixed per our discussion